packages/editor/src/client/panel/publishArticlePanel.tsx

```diff
@@ -12,6 +12,7 @@ import {DateTimePicker} from '../atoms/dateTimePicker'
 import {InfoColor} from '../atoms/infoMessage'
 
 export interface PublishArticlePanelProps {
+  url?: string | null
```

Comment:
it shouldn't be optional. It's always a string but sometimes it's an empty string.